openeo/Sentinel-3.ipynb

```diff
@@ -0,0 +1,444 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "21ec5555",
+   "metadata": {},
+   "source": [
+    "# Sentinel-3\n",
+    "\n",
+    "\n",
+    "This notebook explores working with Sentinel-3 data. \n",
+    "\n",
+    "## Sentinel-3 OLCI\n",
+    "\n",
+    "https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-3-olci\n",
+    "\n",
+    "The OLCI dataset provided by Sentinelhub is based on the level-1b products. These products are provided in \"instrument\" projection rather than being projected into a ground-based reference system. Hence, these products do not come with a 'native' reference system. The openEO layer is currently configured to use EPSG:4326 unprojected coordinates, with a resolution set to a fixed value that tries to approximate the native 300m ground resolution.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7767f70b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openeo\n",
+    "import xarray\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "conn = openeo.connect(\"openeo.dataspace.copernicus.eu\")"
```

Comment:
Usually it's recommended to put imports and real  logic like  `openeo.connect()`  here in separate notebook cells

Reply:
updated